packages/core/components/RichTextEditor/styles.module.css

```diff
@@ -0,0 +1,86 @@
+.Editor-button-group {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 12px;
+  border: 1px solid var(--puck-color-grey-09);
+  border-radius: 4px;
+  background-color: var(--puck-color-white);
+  padding: 4px;
+  margin-bottom: 16px;
+}
+
+.Editor-editor-menu-alignment,
+.Editor-editor-menu-text,
+.Editor-editor-menu-blocks,
+.Editor-editor-menu-breaks {
+  display: flex;
+  align-items: space-between;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 4px;
+  z-index: 3;
+  position: relative;
+}
+
+.Input-input {
+  background: var(--puck-color-white);
+  border-width: 1px;
+  border-style: solid;
+  border-color: var(--puck-color-grey-09);
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-family: inherit;
+  font-size: 12px;
+  padding: 4px 8px;
+  padding-right: 20px;
+  transition: border-color 50ms ease-in;
+}
+
+select.Input-input {
+  appearance: none; /* Safari */
+  background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='100' height='100' fill='%23c3c3c3'><polygon points='0,0 100,0 50,50'/></svg>")
+    no-repeat;
+  background-size: 12px;
+  background-position: calc(100% - 4px) calc(50% + 3px);
+  background-repeat: no-repeat;
+  background-color: var(--puck-color-white);
+  cursor: pointer;
+}
+
+select.Input-input:dir(rtl) {
+  background-position: 12px calc(50% + 3px);
+}
+
+@media (hover: hover) and (pointer: fine) {
+  .Input:has(> input):hover .Input-input:not([readonly]),
+  .Input:has(> textarea):hover .Input-input:not([readonly]) {
+    border-color: var(--puck-color-grey-05);
+    transition: none;
+  }
+  .Input:has(> select):hover .Input-input:not([disabled]) {
+    background-color: var(--puck-color-azure-12);
+    background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='100' height='100' fill='%235a5a5a'><polygon points='0,0 100,0 50,50'/></svg>");
+    border-color: var(--puck-color-grey-05);
+    transition: none;
+  }
+}
+
+.Input-input:focus {
+  border-color: var(--puck-color-grey-05);
+  outline: 2px solid var(--puck-color-azure-05);
+  transition: none;
+}
+
+.Input--readOnly > .Input-input,
+.Input--readOnly > select.Input-input {
+  background-color: var(--puck-color-grey-11);
+  border-color: var(--puck-color-grey-09);
+  color: var(--puck-color-grey-04);
+  cursor: default;
+  opacity: 1;
+  outline: 0;
+  transition: none;
+}
```

Comment:
I didn't expect to see these `.Input` styles here, as they impact a component outside of the `RichTextEditor` scope. Generally, the styles are co-located with the component of the same name (although, the line is a little blurry with fields).

Are these changes specifically for the RTE?

Reply:
This is all for the heading selector dropdown

Reply:
I've moved to BlockStyleSelect